twilight-validate/src/component/component_v2.rs

```diff
@@ -355,6 +396,46 @@ fn label_description(value: impl AsRef<str>) -> Result<(), ComponentValidationEr
     }
 }
 
+/// Validate a [`FileUpload::max_values`] amount.
+///
+/// # Errors
+///
+/// Returns an error of type [`FileUploadMaximumValuesCount`] if the provided number
+/// of values that can be chosen is larger than
+/// [the maximum][`FILE_UPLOAD_MAXIMUM_VALUES_LIMIT`].
+///
+/// [`FileUpload::max_values`]: twilight_model::channel::message::component::FileUpload::max_values
+/// [`FileUploadMaximumValuesCount`]: ComponentValidationErrorType::FileUploadMaximumValuesCount
+const fn component_file_upload_max_values(count: u8) -> Result<(), ComponentValidationError> {
+    if count > FILE_UPLOAD_MAXIMUM_VALUES_LIMIT {
+        return Err(ComponentValidationError {
+            kind: ComponentValidationErrorType::FileUploadMaximumValuesCount { count },
+        });
+    }
+
+    Ok(())
+}
+
+/// Validate a [`FileUpload::min_values`] amount.
+///
+/// # Errors
+///
+/// Returns an error of type [`FileUploadMinimumValuesCount`] if the provided number
+/// of values that must be chosen is larger than
+/// [the maximum][`FILE_UPLOAD_MINIMUM_VALUES_LIMIT`].
+///
+/// [`FileUpload::min_values`]: twilight_model::channel::message::component::FileUpload::min_values
+/// [`FileUploadMinimumValuesCount`]: ComponentValidationErrorType::FileUploadMinimumValuesCount
+const fn component_file_upload_min_values(count: u8) -> Result<(), ComponentValidationError> {
+    if count > FILE_UPLOAD_MINIMUM_VALUES_LIMIT {
```

Comment:
```suggestion
    if count < FILE_UPLOAD_MINIMUM_VALUES_LIMIT {
```

Reply:
Since this is zero `clippy` will probably shout at this so it can probably just be removed.